.github/workflows/verifyimage.yml

```
@@ -107,3 +108,41 @@ jobs:
             grep -q "Access-Control-Allow-Methods: GET, POST, PUT, DELETE, OPTIONS" headers.txt
             grep -q "Access-Control-Allow-Headers: *" headers.txt
           fi
+
+      - name: Checkout CRS
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+          repository: coreruleset/coreruleset
+          path: crs
+      - name: "Install go-ftw"
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          cd crs
+          gh release download -R coreruleset/go-ftw "v${{ env.GO_FTW_VERSION }}" \
+            -p "ftw_${{ env.GO_FTW_VERSION }}_linux_amd64.tar.gz" -O - | tar -xzvf - ftw
+      - name: Patch CRS compose file to use verification image
+        run: |
+          sed -i \
+            's#image: owasp/modsecurity-crs:${{ contains(matrix.target, 'apache') && 'apache' || 'nginx' }}.*#image: ${{ matrix.target }}-verification#' \
+            crs/tests/docker-compose.yml
+      - name: Run CRS tests for ${{ matrix.target }}
+        run: |
+          cd crs
+          modsec_version="${{ contains(matrix.target, 'apache') && 'modsec2-apache' || 'modsec3-nginx' }}"
+          backend="${{ contains(matrix.target, 'apache') && 'httpd' || 'nginx' }}"
+          mkdir -p "tests/logs/${modsec_version}/{nginx,apache2}"
+          docker compose -f ./tests/docker-compose.yml up -d "${modsec_version}"
+          docker compose -f ./tests/docker-compose.yml logs
```

Perhaps the reason the go-ftw test is failing with the alpine image is that the test begins before nginx has started. 
It might be fixed to wait for nginx to start by using sleep etc.

```suggestion
          docker compose -f ./tests/docker-compose.yml up -d "${modsec_version}"
          sleep 30
          docker compose -f ./tests/docker-compose.yml logs
```

---

`up` is synchronous. It shouldn't be necessary to wait.

---

It's definitely a segfault.